docs/source/docs/integration/advancedStrategies.md

```diff
@@ -12,30 +16,49 @@ WPILib provides a [Pose2d](https://docs.wpilib.org/en/stable/docs/software/advan
   \- Sufficient sensors to measure wheel rotation
   \- Capable of closed-loop velocity control
 - A gyroscope or IMU measuring actual robot heading
-- Experience using some path-planning library (WPILib is our recommendation)
+- Experience using some path-planning library
+
+## Robot Poses from the Camera
+
+When using 3D mode in PhotonVision, an additional step is run to estimate the 3D position of camera, relative to one or more AprilTags. These must be filtered and transformed before they represent an estimated robot pose.
```

Comment:
This sections feels unclear, but I can't exactly articulate how at the moment. I'll take another crack at it in the morning. I believe it is something about the details that are included and ordering.

Reply:
I think this sentence is dumb. I made it better. Maybe.